docs/userguide/install.rst

```diff
@@ -12,11 +12,14 @@ Installation
 Install pre-built packages
 ===========================
 
-Download pre-built packages from `ROCm's package servers <https://docs.amd.com/bundle/ROCm-Installation-Guide-v5.4.3/page/Introduction_to_ROCm_Installation_Guide_for_Linux.html>`_. Release notes
-are available on the releases tab of the `library's github page <https://github.com/ROCmSoftwarePlatform/hipSOLVER>`_.
+Download pre-built packages from `ROCm's package servers <https://docs.amd.com/bundle/ROCm-Installation-Guide-v5.4.3/page/Introduction_to_ROCm_Installation_Guide_for_Linux.html>`_.
+Release notes are available on the releases tab of the `library's github page <https://github.com/ROCmSoftwarePlatform/hipSOLVER>`_.
 
 * `sudo apt update && sudo apt install hipsolver`
 
+Note that the pre-built packages depend on the third-party library SuiteSparse, which must be installed on the system prior to installing hipSOLVER.
+SuiteSparse can be installed using the package manager of most distros.
```

Comment:
Yeah, that's clearer now. 
Should we also add a comment like:
"For now, if a user does not need the sparse functionality and does not want to install suitsparse, it is recommended to build hipSOLVER from source" ?